federation/values.yaml

```
@@ -20,3 +20,22 @@ fedsvc:
   service:
     port: 9500
     type: LoadBalancer
+# Atlas Cluster configuration for ObjectScale persistent Storage
+atlas:
+
+  # Disables the affinity rules for the Atlas Cluster, allowing it to run
+  # multiple replicas on a single node
+  disableAntiAffinity: false
+
+  # Configure the container image
+  image:
+    pullPolicy: IfNotPresent
+    repository: atlas
+    tag: v1.1.2
```

needs to be the latest atlas. 

```suggestion
    tag: v1.1.3
```

---

Checked iamsvc - value is 1.1.3 (no longer v1).  Made the change here to match.